library/bonita

```diff
@@ -1,7 +1,11 @@
 Maintainers: Jérémy Jacquier-Roux <jeremy.jacquier-roux@bonitasoft.org> (@JeremJR),
-             Guillaume Rosinosky <guillaume.rosinosky@bonitasoft.org> (@guillaumerosinosky),
              Truc Nguyen <truc.nguyen@bonitasoft.org> (@tnguyen1),
              Laurent Leseigneur <laurent.leseigneur@bonitasoft.org> (@laurentleseigneur)
+             Baptiste Mesta <baptiste.mesta@bonitasoft.org> (@baptistemesta)
+             Danila Mazour <danila.mazour@bonitasoft.org> (@danila-m)
+             Emmanuel Duchastenier <emmanuel.duchastenier@bonitasoft.org> (@educhastenier)
+             Pascal Garcia <pascal.garcia@bonitasoft.org> (@passga)
+
```

Comment:
Couple minor issues -- this blank line shouldn't be here, and `Maintainers:` needs commas on the list items.

(Otherwise this is great :+1:)

Reply:
(and if you'd prefer, I'm happy to push directly to the branch to fix the formatting issues)

Reply:
sorry, it should be better now.
thank you